WcaOnRails/app/models/round.rb

```diff
@@ -40,6 +40,40 @@ def event
     Event.c_find(competition_event.event_id)
   end
 
+  # Compute a round type id from round information
+  def round_type_id
+    if number == total_number_of_rounds
+      if cutoff
```

Comment:
Nit: I think this whole function would get a lot shorter (and therefore more readable) if you use ternaries here. I'm thinking something like:

```ruby
cutoff ? "c" : "f"
```

That gets 5 lines of code down to 1 line, and should make this whole function a lot shorter.